charts/tidb-lightning/values.yaml

```diff
@@ -95,9 +95,17 @@ backend: importer # importer | tidb
 # Specify a Service Account for lightning
 # serviceAccount:
 
-# For TiDB-Lightning v3.0.18+ or v4.0.3+,
-# if we want to log to stdout, we should set `file = "-"`.
+# For TiDB-Lightning v3.0.18+ or v4.0.3+, if you want to log to stdout, you should set `file = "-"`.
+# If you want to store the checkpoint under source data directory, you should set `CHECKPOINT_USE_DATA_DIR` as the prefix of `dsn`.
```

Comment:
```suggestion
# If you want to store the checkpoint under source data directory, you should set `CHECKPOINT_USE_DATA_DIR` as the prefix of `dsn`.
# If you do not update the `CHECKPOINT_USE_DATA_DIR` in the following `dsn` field, it will be replaced automatically in the startup script with the following rules:
```